cartography/stats.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, client: StatsClient, prefix: str = None):
         self._client = client
         self._scope_prefix = prefix
 
-    def get_scoped_stats_client(self, scope: str):
+    def get_stats(self, scope: str):
```

Comment:
I think this name should be more specific to what is returned. "get_stats_client" is more descriptive of what it's doing. Please add a type hint for the return object.

Reply:
Updated. Added the type hint as a string as well (https://stackoverflow.com/questions/33533148/how-do-i-type-hint-a-method-with-the-type-of-the-enclosing-class)